.github/workflows/linters_and_formatters.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Linters and Formatters
+
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
I think we should run this on all branches since it's not as expensive as building a Docker container and we can detect problems early on. So instead of `'*'` we can use `'**'`.

Reply:
Oh good to know the diff between * and ** lol
Tha